superdesk/wsgi.py

```diff
@@ -0,0 +1,11 @@
+"""WSGI entrypoint.
+
+Usage via Procfile:
+
+    rest: gunicorn -c pythong:superdesk.gunicorn_config superdesk.wsgi
+
+"""
+
+from superdesk.factory import get_app
+
+application = get_app()
```

Comment:
I think it could be combined with `superdesk.worker`, probably in module `superdesk.honcho` and usage like
```
rest: gunicorn -c python:superdesk.gunicorn_config superdesk.honcho
work: celery -A superdesk.honcho worker
beat: celery -A superdesk.honcho beat --pid=
```